### PR TITLE
ci(package.json): fix cypress on forks

### PR DESCRIPTION
**Pull request by ArturBa**
#770

### CODE REVIEW

File: workspaces/web/percypress.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+if [ "$KNAPSACK_PRO_TEST_SUITE_TOKEN_RSPEC" = "" ]; then
+  KNAPSACK_PRO_ENDPOINT=https://api-disabled-for-fork.knapsackpro.com \
+    KNAPSACK_PRO_TEST_SUITE_TOKEN_RSPEC=disabled-for-fork \
+    KNAPSACK_PRO_MAX_REQUEST_RETRIES=0 
+fi
+KNAPSACK_PRO_TEST_FILE_PATTERN=\"cypress/integration/**/*.{feature,features}\" percy exec -- $(npm bin)/knapsack-pro-cypress
```
Comment (kantord) [inline workspaces/web/percypress.sh:7]: I think when the token is not set, it should be just running `cypress run`, no?

Comment (ArturBa) [reply workspaces/web/percypress.sh]: Fixed

**Comment by kantord** [comment]
:100: 
Thank you @ArturBa !!!

**Comment by kantord** [comment]
@allcontributors please add @ArturBa for code

**Pull request merged**